_source/_posts/2017-12-04-use-kong-gateway-to-centralize-authentication.md

```diff
@@ -8,6 +8,8 @@ tags: [kong, api-gateway, api, gateway, spring, spring-boot]
 type: conversion
 ---
 
+> **NOTE:** This post has just been updated for 2021!
```

Comment:
I don't see any updates to the post. Maybe it'd be better to say the video? 

Related: I plan to add "last updated" functionality. It seems like it's possible. https://stackoverflow.com/questions/36758072/how-to-insert-the-last-updated-time-stamp-in-jekyll-page-at-build-time

Reply:
updated